composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "leandrogehlen/yii2-treegrid",
+    "name": "nikserg/yii2-treegrid",
```

Comment:
Is not possible change vendor name

Reply:
Fixed in https://github.com/leandrogehlen/yii2-treegrid/pull/29